sw/blas/gemm/src/gemm.h

```diff
@@ -5,11 +5,13 @@
 // Author: Tim Fischer <fischeti@iis.ee.ethz.ch>
 //         Luca Bertaccini <lbertaccini@iis.ee.ethz.ch>
 //         Luca Colagrande <colluca@iis.ee.ethz.ch>
+//         Viviane Potocnik <vivianep@iis.ee.ethz.ch>
 
 #include <stdint.h>
 
 #include "snrt.h"
 
+#define BASELINE 1
```

Comment:
My suggestion would be to define another parameter in the `param.hjson` file for this instead of using defines. This would make profiling of kernels easier. I started this in another PR https://github.com/pulp-platform/snitch_cluster/pull/83